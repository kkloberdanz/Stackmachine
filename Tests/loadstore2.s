PUSH
10
STORE
0
PUSH
11
STORE
1
PUSH
12
STORE
2
LOAD
1
LOAD
0
LOAD
2
ADD
ADD
HALT
