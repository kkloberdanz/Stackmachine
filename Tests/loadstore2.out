1
10
13
0
1
11
13
1
1
12
13
2
12
1
12
0
12
2
2
2
999999
